clippy_lints/src/literal_representation.rs

```diff
@@ -211,11 +232,18 @@ impl<'a> DigitInfo<'a> {
             if self.radix == Radix::Hexadecimal && nb_digits_to_fill != 0 {
                 hint = format!("{:0>4}{}", &hint[..nb_digits_to_fill], &hint[nb_digits_to_fill..]);
             }
+            let suffix_hint = match self.suffix {
+                Some(suffix) if ["_8", "_32", "_64"].contains(&suffix) => {
+                    format!("_i{}` or `{}_u{}", &suffix[1..], hint, &suffix[1..])
```

Comment:
Didn't know that rustfix used these suggestions like that. 

Do we want this lint to be automatically applied? With support for suggesting `_i128` instead of `_128` we can end up with false positives, since `_128` can be part of a properly formatted literal.

Reply:
Oh your right. `256_128` is a valid formatting for a int literal and would even be suggested by one of those lints for `256128`, which would then be linted to change it to `256_i128`. I have no idea how to handle this. We can't really know what to do with `_128` and I would suggest to just bail out and write this in the `Known Problems` section.

If you come up with a good idea, you can add it of course! If not just leave out the `_128` case.